### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**
Removes title case, 'factory reset' over 'reset', and 'mnemonic seed'
over 'bip39 mnemonic'.

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -18,31 +18,31 @@
   },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Missing above:
- account.exportTransactions
- account. openFile

File: frontends/web/src/locales/en/app.json [hunk 2]
```diff
@@ -18,31 +18,31 @@
...
   "accountInfo": {
     "address": "Address",
-    "extendedPublicKey": "Extended Public Key",
+    "extendedPublicKey": "Extended public key",
     "title": "Account Information",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -18,31 +18,31 @@
...
     "error": {
       "alreadyExists": "The account already exists.",
       "invalidAddress": "Please enter a valid address",
       "xprivEntered": "WARNING: You entered an extended private key. Please enter an extended public key and keep your private keys safe.",
-      "xpubInvalid": "Extended Public Key malformatted."
+      "xpubInvalid": "Extended public key malformatted."
     },
-    "extendedPublicKey": "Extended Public Key",
+    "extendedPublicKey": "Extended public key",
     "scriptType": "Script Type",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 4]
```diff
@@ -18,31 +18,31 @@
...
     "submit": "Add Account",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 5]
```diff
@@ -18,31 +18,31 @@
...
     "title": "Add Account",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 6]
```diff
@@ -76,15 +76,15 @@
         "placeholder": "Please name the backup"
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: name.label one above

File: frontends/web/src/locales/en/app.json [hunk 7]
```diff
@@ -76,15 +76,15 @@
...
       "verificationFailed": "The recovery password does NOT MATCH the current wallet. The backup has been created. Please use 'Check Backup' to verify your recovery password again."
     },
     "description": "Select <strong>wallet backup file</strong>",
-    "insert": "Please insert the micro SD card to manage backups.",
-    "insertButton": "I have inserted the micro SD card",
+    "insert": "Please insert the microSD card to manage backups.",
+    "insertButton": "I have inserted the microSD card",
     "list": "Your microSD card backups",
     "noBackups": "There are no backups on this microSD card.",
     "restore": {
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: confirmTitle one below

Comment (KasparEtter) [reply frontends/web/src/locales/en/app.json]: mabe in e200: microSD card not found

File: frontends/web/src/locales/en/app.json [hunk 8]
```diff
@@ -103,13 +103,13 @@
...
     "showMnemonic": {
       "description": "You will be presented with up to 24 words, which form a backup of your wallet. Write them down on paper.\n\n<strong>Do not store them digitally or take pictures of it.</strong>\n\n<strong>Do not say the words out loud.</strong>\n\n<strong>This backup is not password-protected.</strong>\n\nAfterwards, you will be asked to confirm each word.",
-      "title": "Show BIP39 Mnemonic"
+      "title": "Show mnemonic seed"
     },
-    "title": "Manage Backups"
+    "title": "Manage backups"
   },
   "bb02Bootloader": {
     "abort": "Don't upgrade – Take me back",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: `Don't upgrade, take me back`

File: frontends/web/src/locales/en/app.json [hunk 9]
```diff
@@ -163,7 +163,7 @@
     "create": {
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Missing above:
- Advanced settings
- Current device name
- BitBox02 name

File: frontends/web/src/locales/en/app.json [hunk 10]
```diff
@@ -163,7 +163,7 @@
...
       "button": "Name Device & Continue",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 11]
```diff
@@ -185,20 +185,20 @@
       "unpaired": "A new BitBox02 has been detected. Please verify that the following code matches what is shown on your device. If the code matches, touch underneath the check mark on your BitBox02 and then click the button below to continue."
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: One above: Verify pairing code

File: frontends/web/src/locales/en/app.json [hunk 12]
```diff
@@ -211,13 +211,13 @@
       "title": "Choose your syncing option"
     },
     "pairing": {
-      "failed": "Unconfirmed pairing. Please try to disconnect and connect to the BitBox Base again",
-      "paired": "You have confirmed on your BitBox Base that the code matches",
+      "failed": "Unconfirmed pairing. Please try to disconnect and connect to the BitBoxBase again",
+      "paired": "You have confirmed on your BitBoxBase that the code matches",
       "title": "Verify Pairing Code",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 13]
```diff
@@ -286,7 +286,7 @@
     "hardware": {
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above:
- Enable two factor authorization (2FA) [several times]
- Available version

File: frontends/web/src/locales/en/app.json [hunk 14]
```diff
@@ -300,7 +300,7 @@
       },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: 2FA again

Comment (KasparEtter) [reply frontends/web/src/locales/en/app.json]: two lines above

File: frontends/web/src/locales/en/app.json [hunk 15]
```diff
@@ -300,7 +300,7 @@
...
       "mobile": {
         "false": "Closed",
-        "label": "Mobile App",
+        "label": "Mobile app",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: I'm not sure where this is used but maybe we should more consistently start using "mobile verification app" (or something like this) in preparation for the Android release of this app.

File: frontends/web/src/locales/en/app.json [hunk 16]
```diff
@@ -425,15 +425,15 @@
     },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above:
- App version
- Security information

File: frontends/web/src/locales/en/app.json [hunk 17]
```diff
@@ -611,7 +611,7 @@
       },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above:
- Segregated witness benefits
- Backup center

File: frontends/web/src/locales/en/app.json [hunk 18]
```diff
@@ -668,18 +668,18 @@
       "text": "The pairing has been aborted from the mobile app.",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above:
- Disable legacy hidden wallet
- Enable legacy hidden wallet

File: frontends/web/src/locales/en/app.json [hunk 19]
```diff
@@ -668,18 +668,18 @@
...
     "error": {
       "text": "Something went wrong. Please start again.",
       "title": "Error"
     },
     "reconnectOnly": {
-      "button": "Reconnect Mobile App"
+      "button": "Reconnect mobile app"
     },
     "start": {
       "hideAppQRCode": "Hide QR Code",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: code (also one below)

File: frontends/web/src/locales/en/app.json [hunk 20]
```diff
@@ -709,7 +709,7 @@
     }
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above: Mobile pairing

File: frontends/web/src/locales/en/app.json [hunk 21]
```diff
@@ -747,12 +747,12 @@
...
+      "description5": "The wallet you create will be backed up to a file on the microSD card provided. This can be used in an emergency to recover your funds using your <strong>recovery password</strong>.",
       "title": "Security Information"
     },
     "restore": {
       "button": "Continue",
-      "description1": "You will be asked to insert the micro SD card you used to store you backup.",
+      "description1": "You will be asked to insert the microSD card you used to store you backup.",
       "description2": "To restore your device you will need your recovery password. ",
       "description3": "Please take care when entering the recovery password. Any password you enter will create a valid wallet. If you enter the wrong password you may be shown a wallet balance that you do not expect.",
       "title": "Security Information"
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 22]
```diff
@@ -747,12 +747,12 @@
       "description2": "You will be asked to create two passwords.",
       "description3": "The first is the <strong>device password</strong> which unlocks your BitBox device and can be changed later.",
       "description4": "The second is the <strong>recovery password</strong> which unlocks your wallet. This password cannot be changed later.",
-      "description5": "The wallet you create will be backed up to a file on the micro SD card provided. This can be used in an emergency to recover your funds using your <strong>recovery password</strong>.",
+      "description5": "The wallet you create will be backed up to a file on the microSD card provided. This can be used in an emergency to recover your funds using your <strong>recovery password</strong>.",
       "title": "Security Information"
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 23]
```diff
@@ -764,18 +764,18 @@
...
     "description": "I understand that:",
     "error": {
       "e102": "The password must consist of at least 8 characters.",
-      "e200": "You need to insert a micro SD card into your BitBox in order to create a wallet, so a backup can be created automatically."
+      "e200": "You need to insert a microSD card into your BitBox in order to create a wallet, so a backup can be created automatically."
     },
     "info": {
       "button": "Set recovery password now",
-      "description1": "Insert the micro SD card in to the BitBox",
+      "description1": "Insert the microSD card in to the BitBox",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: into (instead of "in to")?

File: frontends/web/src/locales/en/app.json [hunk 24]
```diff
@@ -789,11 +789,11 @@
   },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above: Wallet name

File: frontends/web/src/locales/en/app.json [hunk 25]
```diff
@@ -814,7 +814,7 @@
     "coincontrol": {
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above: Sign and send

File: frontends/web/src/locales/en/app.json [hunk 26]
```diff
@@ -814,7 +814,7 @@
...
       "address": "Address",
       "outpoint": "Outpoint",
-      "title": "Send from Output"
+      "title": "Send from output"
     },
     "confirm": {
       "selected-coins": "Selected Coins",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

And below:
- Confirm transaction
- Converted fee (twice)
- Optional data

File: frontends/web/src/locales/en/app.json [hunk 27]
```diff
@@ -864,7 +864,7 @@
     },
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above: Scan QR code

File: frontends/web/src/locales/en/app.json [hunk 28]
```diff
@@ -897,36 +897,36 @@
...
       "useProxy": "Enable tor proxy"
     },
-    "restart": "Please re-start the BitBox app for the changes to take effect.",
+    "restart": "Please re-start the BitBoxApp for the changes to take effect.",
     "success": "Please unplug and replug the BitBox for the changes to take effect.",
     "title": "Settings"
   },
   "setup": "Setup device",
   "sidebar": {
     "addAccount": "Add Account",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: ⚠️

File: frontends/web/src/locales/en/app.json [hunk 29]
```diff
@@ -984,9 +984,9 @@
     "unlocked3": "Tap the touch button when the LED lights up"
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: Above:
- Get started
- Open in external block explorer
- Pending transaction
- Upgrade firmware (twice)
- Do you want to upgrade the firmware…

File: frontends/web/src/locales/en/app.json (hunk 15 above)
Comment (benma) [reply frontends/web/src/locales/en/app.json]: won't do now as i'd have to check if it fits the UI

**Pull request merged**